code/cfile/cfilesystem.cpp

```diff
@@ -592,9 +592,31 @@ void cf_build_root_list(const char *cdrom_dir)
 	if ( !_getcwd(working_directory, CF_MAX_PATHNAME_LENGTH ) ) {
 		Error(LOCATION, "Can't get current working directory -- %d", errno );
 	}
+	if (Cmdline_override_data) {
```

Comment:
Some spacing before and after this block would be preferred. Less likely to be confused as being part of the `if` statement above here.